manifests/fpm.pp

```diff
@@ -98,7 +98,7 @@
 
   # Create an override to use a reload signal as trusty and utopic's
   # upstart version supports this
-  if $::operatingsystem == 'Ubuntu' and ($::operatingsystemmajrelease == '14.04' or $::operatingsystemmajrelease == '14.10') {
+  if $facts['os']['name'] == 'Ubuntu' and ($facts['os']['release']['major'] == '14') {
```

Comment:
Turns out 'major' is actually '14.04' not just '14'. https://github.com/camptocamp/facterdb/blob/54422fbe9e32cfacddae5403237ee2758a7f26af/facts/3.7/ubuntu-14.04-x86_64.facts#L339